Open Judge System/Workers/OJS.Workers.ExecutionStrategies/JsonExecutionResult.cs

```diff
@@ -20,6 +20,10 @@ public class JsonExecutionResult
 
         public List<int> PassingIndexes { get; set; }
 
```

Comment:
Rather inconsistent property naming. Perhaps include "Tests" in the name, e.g TotalPassedTests / TotalTestPasses, or remove it all together from the other properties as well "TestsErrors" to simply "Tests".